### PR TITLE
chore: downgrade node version required

### PR DESCRIPTION
**Pull request by T-Wizard**
For use in sense-client that only has node 18.15.0 because of https://github.com/nodejs/node/issues/47563

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "sn-scatter-plot-ext"
   ],
   "engines": {
-    "node": ">=v18.16.0"
+    "node": ">=v18.15.0"
   },
```
Comment (T-Wizard) [inline package.json:29]: or should we fully remove required node version?

**Comment by T-Wizard** [comment]
as is this I expect it to only be temporary and that renovate will update it again

**Comment by veinfors** [comment]
> As this is, I expect it to only be temporary and that renovate will update it again

We could get around that by telling renovate not to update "engines" like this:
```
"packageRules": [
  {
    "depTypeList": ["engines"],
    "enabled": false
  }
]
```

As it's done in picasso for instance: https://github.com/qlik-oss/picasso.js/blob/0c9fd35249ae1a911dc77b02b04d99a05c99df43/renovate.json#L7

File: package.json (hunk 1 above)
Comment (veinfors) [reply package.json:29]: Either that, or tell renovate not to update engines...

**Pull request merged**